targets/targets.json

```diff
@@ -2471,7 +2479,8 @@
             "SERIAL_ASYNCH",
             "TRNG",
             "FLASH",
-            "MPU"
+            "MPU",
+            "WATCHDOG"
```

Comment:
This is not needed as target inherits from FAMILY_STM32 where WATCHDOG is already defined